docs/src/_getting-started/part1.md

```diff
@@ -0,0 +1,217 @@
+    <p>If you're having trouble throughout this tutorial, you can always head over to the <a href="https://meltano.com/slack">Slack channel</a> to get help.</p>
+</div>
+
+## Create Your Meltano Project
+Step 1 is to create a new [Meltano project](/concepts/project) that (among other things)
+will hold the [plugins](/concepts/plugins) that implement the details of our E(t)LT pipeline.
+
+
+1. Navigate to the directory that you'd like to hold your Meltano projects.
+
+2. Initialize a new project in a directory of your choosing using [`meltano init`](/reference/command-line-interface#init):
+
+   ```bash
+   meltano init my-meltano-project
+   ```
+
+![Meltano Init](images/part1/gif_meltano_init.gif)
+
+   This action will create a new directory with, among other things, your [`meltano.yml` project file](/concepts/project#meltano-yml-project-file). Your file will look something like this:
+
+   ```yml
+   version: 1
+   default_environment: dev
+   project_id: u3u5u4ueudufhfh4h4h5ddkjh
+   environments:
+   - name: dev
+   - name: staging
+   - name: prod
+   ```
+
+1. Navigate to the newly created project directory:
+
+   ```bash
+   cd my-meltano-project
+   ```
+
+## Add an Extractor to Pull Data from a Source
+
+Now that you have your very own Meltano project, it's time to add [plugins](/concepts/plugins) to it. We're going to add an extrator for GitLab to get our data. An [extractor](/concepts/plugins#extractors) is responsible for pulling data out of any data source.
+
+1.  Add the GitLab extractor
+
+    ```bash
+    $ meltano add extractor tap-gitlab
+    ```
+![Meltano Version](images/part1/gif_meltano_add_extr.gif)
+
+    This will add the new plugin to your [`meltano.yml` project file](/concepts/project#plugins):
+
+    ```yml
+    plugins:
+    extractors:
+      - name: tap-gitlab
+        variant: meltanolabs
+        pip_url: git+https://github.com/MeltanoLabs/tap-gitlab.git
+    ```
+
+
+1.  Test that the installation was successful by calling [`meltano invoke`](/reference/command-line-interface#invoke):
+
+    ```bash
+    $ meltano invoke tap-gitlab --help
+    ```
+    If you see the extractor's help message printed, the plugin was definitely installed successfully.
+    ![Meltano Tap Help](images/part1/gif_meltano_invoke_tg_help.gif)
+
+
+### Configure the Extractor
+
+The GitLab tap requires [configuration](/guide/configuration) before it can start extracting data.
+
+1. The simplest way to configure a new plugin in Meltano is using the mode `interactive`:
+
+   ```bash
+   $ meltano config tap-gitlab set --interactive
+   ```
+2. Follow the prompts to step through all available settings, the ones you'll need to fill out are projects, start_date and your private_token.
+
+![Meltano Version](images/part1/gif_meltano_config_interact.gif)
+
+  This will add the non-sensitive configuration to your [`meltano.yml` project file](/concepts/project#plugin-configuration):
+
+  ```yml
+  environments:
```

Comment:
By default now, `meltano config` does _not_ use the default environment. The config will just be under the top level `plugins` key. Documented in https://docs.meltano.com/concepts/environments#default-environments

Reply:
Which part are you commenting on? I get that I need to change something, but not sure what :-D I do want to keep out the environment config part in part 1 of the tut if possible.

Reply:
found it.